### DISCUSSION TITLE
Turkish Characters

### DISCUSSION THREAD

**Discussion by Yakrel** [siweia/NDui#178] [Q&A] [answered]
Hello when I write Turkish characters ingame like "ğ ö ı ç " game shows these characters as blocks is there any setting to fix this ? I tried to find but I couldn't.

**Comment by siweia** [comment]
It means the font you use for game, doesn't support the Turkish you input. You only need to replace fonts in you game fonts file.

**Comment by Yakrel** [comment]
I am not changing font or any setting I am using default NDui  and this problem occurs, but when when NDui disabled I don't have this problem so is this problem not about NDui default font ? Cant you somehow add Turkish characters to NDui font ?

**Comment by siweia** [comment, marked as answer]
There aren't any fonts in NDui. You might need to check on the option in ChatFrame, which uses the default font for chat msg.
